verbs compilation failure as of #1334

Looks like #1334 broke the verbs provider build.  From the Cisco Jenkins log:
```
13:50:37   CC       prov/verbs/src/src_libfabric_la-fi_verbs.lo
13:50:37 prov/verbs/src/fi_verbs.c: In function 'fi_ibv_eq_open':
13:50:37 prov/verbs/src/fi_verbs.c:2835:56: error: 'struct dlistfd_head' has no member named 'fd'
13:50:37   if (epoll_ctl(_eq->epfd, EPOLL_CTL_ADD, _eq->list_head.fd[LIST_READ_FD], &event)) {
13:50:37                                                         ^
13:50:37 prov/verbs/src/fi_verbs.c:2835:60: error: 'LIST_READ_FD' undeclared (first use in this function)
13:50:37   if (epoll_ctl(_eq->epfd, EPOLL_CTL_ADD, _eq->list_head.fd[LIST_READ_FD], &event)) {
13:50:37                                                             ^
13:50:37 prov/verbs/src/fi_verbs.c:2835:60: note: each undeclared identifier is reported only once for each function it appears in
13:50:37 make[1]: *** [prov/verbs/src/src_libfabric_la-fi_verbs.lo] Error 1
13:50:37 make[1]: *** Waiting for unfinished jobs....
13:50:38 make[1]: Leaving directory `/home/jenkins/slave-roots/blade-jenkins1/workspace/libfabric-sanity/libfabric'
13:50:38 make: *** [all] Error 2
```

@shefty will you take a look?

--- a/prov/verbs/src/fi_verbs.c
+++ b/prov/verbs/src/fi_verbs.c
@@ -66,6 +66,7 @@
 #include <rdma/fi_log.h>
 #include "prov.h"
 #include "fi_list.h"
+#include "fi_signal.h"
 
 
 static int fi_ibv_getinfo(uint32_t version, const char *node, const char *service,
@@ -2832,7 +2833,8 @@ fi_ibv_eq_open(struct fid_fabric *fabric, struct fi_eq_attr *attr,
 	memset(&event, 0, sizeof(event));
 	event.events = EPOLLIN;
 
-	if (epoll_ctl(_eq->epfd, EPOLL_CTL_ADD, _eq->list_head.fd[LIST_READ_FD], &event)) {
+	if (epoll_ctl(_eq->epfd, EPOLL_CTL_ADD,
+		      _eq->list_head.signal.fd[FI_READ_FD], &event)) {
 		ret = -errno;
 		goto err3;
 	}

